tmt/steps/prepare/install.py

```diff
@@ -165,17 +165,25 @@ def go(self, guest):
         packages = self.get('package', [])
         repo_packages = []
         local_packages = []
+        repo_debuginfos = []
         directories = self.get('directory', [])
         if not packages and not directories:
             self.debug("No packages for installation found.", level=3)
             return
 
         # Detect local packages and directories
+        # and split debug{info,source} if installing from repo
         for package in packages:
             if package.endswith('.rpm'):
                 local_packages.append(package)
             else:
-                repo_packages.append(package)
+                if 'debuginfo' in package or 'debugsource' in package:
+                    # strip -debuginfo from package name
+                    # installing with it doesn't work on RHEL7
+                    package.replace('-debuginfo','')
```

Comment:
Replace does not work in-place, it returns the modified string. So the original package name is stored instead.

Reply:
Fixed, thanks for the catch.